src/Microsoft.ML.HalLearners/OlsLinearRegression.cs

```diff
@@ -87,9 +89,6 @@ internal OlsLinearRegressionTrainer(IHostEnvironment env, Options options)
         protected override RegressionPredictionTransformer<OlsLinearRegressionModelParameters> MakeTransformer(OlsLinearRegressionModelParameters model, DataViewSchema trainSchema)
              => new RegressionPredictionTransformer<OlsLinearRegressionModelParameters>(Host, model, trainSchema, FeatureColumn.Name);
 
-        public RegressionPredictionTransformer<OlsLinearRegressionModelParameters> Train(IDataView trainData, IPredictor initialPredictor = null)
-            => TrainTransformer(trainData, initPredictor: initialPredictor);
```

Comment:
Why are we removing this?

Reply:
Because it's not used at all.

---
In reply to: [257348853](https://github.com/dotnet/machinelearning/pull/2497#discussion_r257348853) [](ancestors = 257348853)